content/docs/start/experiments.md

```diff
@@ -163,8 +241,12 @@ reproducing the result without `dvc exp run`. Let's make it persistent in our
 regular pipeline by committing it in our Git branch:
 
 ```dvc
-$ git add dvc.lock params.yaml prc.json roc.json scores.json
-$ git commit -a -m "Preserve best random forest experiment"
+$ git add dvc.lock params.yaml logs.csv metrics.json
+$ git commit -m "Preserve the experiment with 128 Conv Units"
+
+[main 84d1c8f] Preserve the experiment with 128 Conv Units
+ 4 files changed, 21 insertions(+), 21 deletions(-)
+ rewrite logs.csv (91%)
 ```
 
 ## Sharing experiments
```

Comment:
This whole section needs to be revised or removed, but again this is a separate issue (see https://github.com/iterative/dvc.org/issues/2474).